coderd/workspaceapps/apptest/setup.go

```diff
@@ -25,13 +25,14 @@ import (
 	"github.com/coder/coder/v2/coderd/workspaceapps"
 	"github.com/coder/coder/v2/codersdk"
 	"github.com/coder/coder/v2/codersdk/agentsdk"
+	"github.com/coder/coder/v2/cryptorand"
 	"github.com/coder/coder/v2/provisioner/echo"
 	"github.com/coder/coder/v2/provisionersdk/proto"
 	"github.com/coder/coder/v2/testutil"
 )
 
 const (
-	proxyTestAgentName            = "agent-name"
+	proxyTestAgentName            = "agnt-name"
```

Comment:
Lol did we hit a limit in the test?

Reply:
Yeah we hit the 62 char limit lol, Marcin reverted his PR though so it should be fine now hopefully